resources/type-system/flow-analysis.md

```diff
@@ -393,6 +403,62 @@ If `N` is an expression, and the above rules specify the value to be assigned to
 
 - TODO: redirecting factory and generative constructors.
 
+### Extended Reachability
+
+  As mentioned earlier, it would be sound to define exit(M) = ∅ for all models, but
+  doing so leads to less than optimal user feedback. Consider, for example, the following code:
+
+  ```dart
+    void f(Object o) {
+      throw ...; // Temporary hack (S1)
+      if (o is! int) return; // (S2)
+      print(o.isEven); // (S3)
+      print(o.hashCodee); // (S4)
+    }
+  ```
+
+  Due to the unconditional throw at (S1), `after(S1)` will be considered unreachable,
+  so both branches of the if statement at (S2) will be considered unreachable.
```

Comment:
To make it clear that we're definitely proposing doing this extension, I recommend reworking this text to be in the subjunctive where appropriate, e.g. "...so both branches of the if statement at (S2) **would** be considered unreachable", and similar changes through the rest of this paragraph.

Reply:
Good point. Reworded.